.gitignore

```diff
@@ -1,6 +1,8 @@
 .settings
 .project
 .cproject
+notes.txt
```

Comment:
You shouldn't add personal files to a projects .gitignore, these belong in your [personal .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

Reply:
Thank you everett.  I am still a total newbie at Git.